Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/revocation/model/RevocationEntryCoordinates.kt

```diff
@@ -1,11 +1,18 @@
 package de.rki.coronawarnapp.covidcertificate.revocation.model
 
-// To Do: Implement
-interface RevocationEntryCoordinates {
-
-    enum class Type(type: String) {
-        SIGNATURE("0a"),
-        UCI("0b"),
-        COUNTRYCODEUCI("0c")
-    }
-}
+import com.google.gson.annotations.SerializedName
+import okio.ByteString
+
+data class RevocationEntryCoordinates(
+    /** The hex-encoded KID of a DCC (usually 16 characters) */
+    @SerializedName("kid") val kid: ByteString,
+
+    /** See [RevocationHashType] */
+    @SerializedName("type") val type: RevocationHashType,
+
+    /** A hex-encoded byte representing the first byte of the hash */
+    @SerializedName("x") val x: ByteString,
+
+    /** A hex-encoded byte representing the second byte of the hash */
+    @SerializedName("y") val y: ByteString
+)
```

Comment:
this is calculated from a DCC, I think we can calculate it on the fly and it does not have to be persisted see [specs](https://github.com/corona-warn-app/cwa-app-tech-spec/blob/abd486e134328fe8484cabd78f65657c3d26366d/docs/spec/dcc-revocation-client.md#data-structure-for-revocation-entry-coordinates)

Reply:
So please remove SerializedName I think it is not needed than ;)